NitroxPatcher/Patches/Dynamic/StarshipDoor_OnDoorToggle_Patch.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Reflection;
+using Harmony;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Core;
+using NitroxModel.DataStructures;
+using NitroxModel.DataStructures.GameLogic.Entities.Metadata;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    class StarshipDoor_OnDoorToggle_Patch : NitroxPatch, IDynamicPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(StarshipDoor);
```

Comment:
Minor: Merge L14 & 15 like in the LockDoor patch.

Reply:
I believe unless I am mistaken that this is not possible as that way of doing that is only possible on public methods

Reply:
I mean
```
public static readonly MethodInfo TARGET_METHOD = typeof(StarshipDoor).GetMethod("OnDoorToggle", BindingFlags.Instance | BindingFlags.NonPublic); 

```